firestore/integration_test_internal/src/bundle_test.cc

```diff
@@ -1,3 +1,5 @@
+// Copyright 2021 Google LLC
+
 #include <future>
```

Comment:
Lint warning: `<future> is an unapproved C++11 header.`<hidden value="cpplint-file-comment"></hidden><hidden value="firestore/integration_test_internal/src/bundle_test.cc:3:  <future> is an unapproved C++11 header.  [build/c++11] [5]"></hidden>